.gitignore

```diff
@@ -10,8 +10,6 @@
 # Ignore bundler config.
 /.bundle
 
-/config/database.yml
-
```

Comment:
I wouldn't include this file here. IMO using `ENV` variables to handle sensitive information is enough for this.